extensions/amp-experiment/1.0/mutation-parser.js

```diff
@@ -28,6 +30,34 @@ const MUTATION_TYPES = [
   'childList',
 ];
 
+const SUPPORTED_ATTRIBUTES = {
+  style: value => {
+
+    // Do not allow Important or HTML Comments
+    if (value.match(/(!\s*important|<!--)/g)) {
+      return false;
+    }
+
+    // Allow Color
+    if (value.match(/^color:\s#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/g)) {
+      return true;
+    }
+
+    // Allow Background color
+    if (value.match(/^background-color:\s#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/g)) {
+      return true;
+    }
+
+    return false;
+  },
+  src: value => {
+    return value.match(/^https:\/\//g);
```

Comment:
may want to use `assertHttpsUrl` here and below https://github.com/ampproject/amphtml/blob/f52c37a1005a5403f9d7c55f54d645826a50c313/src/url.js#L297